packages/react-composites/tests/app/chat/FakeAdapterApp.tsx

```diff
@@ -125,8 +125,7 @@ const handleFileUploads = (adapter: ChatAdapter, fileUploads: _MockFileUpload[])
         name: file.name,
         extension: file.extension,
         url: file.url,
-        attachmentType: 'fileSharing',
-        id: ''
+        attachmentType: 'fileSharing'
```

Comment:
should this be `attachmentType: file.attachmentType`?

Reply:
Yep updated